pipeline/outputs/postgresql.md

```diff
@@ -74,15 +75,16 @@ In your main configuration file add the following section:
 
 ```text
 [OUTPUT]
-    Name          pgsql
-    Match         *
-    Host          172.17.0.2
-    Port          5432
-    User          fluentbit
-    Password      YourCrazySecurePassword
-    Database      fluentbit
-    Table         fluentbit
-    Timestamp_Key ts
+    Name                pgsql
+    Match               *
+    Host                172.17.0.2
+    Port                5432
+    User                fluentbit
+    Password            YourCrazySecurePassword
+    Database            fluentbit
+    Table               fluentbit
+    Connection_Options  project=fluentbit
```

Comment:
This example will not work, please provide an example that actually works here

Reply:
@bkayranci please, provide a proper option here so the documentation it's merged with a working example

Reply:
I updated connection_options with the [requiressl](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-SSLMODE:~:text=to%20disable.-,requiressl,-This%20option%20is) option instead of project. 0 is default value on docs.